plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -107,6 +116,27 @@
       - Define external parameter for this user.
     type: bool
     default: no
+  identities:
+    description:
+      - List of identities to be added/updated for this user.
+    type: list
+    elements: dict
+    suboptions:
+      provider:
+        description:
+          - The name of the external identity provider
+        type: str
+      extern_uid:
```

Comment:
```suggestion
      external_uid:
```
Perhaps clearer as `external` ?

Reply:
There has also been some related discussions about naming here: https://github.com/ansible-collections/community.general/pull/2691#discussion_r645117945

Reply:
I'm slightly in favor of renaming, but not that much that I would request a change. @stgrace what do you think?

Reply:
Let's keep this name for now. If someone wants to change it, please create a follow-up PR (must get merged before 3.3.0 is released, i.e. in ~a week).